3 - Assignment/240326 - Rust Basic + Roguelike Game, Week 1/collatz-sequence/src/lib.rs

```diff
@@ -1,6 +1,17 @@
 /// Determine the length of the collatz sequence beginning at `n`.
 pub fn collatz_length(mut n: i32) -> u32 {
-    todo!("Implement this")
+    let mut count = 0;
+    loop {
+        count += 1;
+        if n == 1 {
+            break;
+        } else if n % 2 == 0 {
+            n /= 2;
+        } else {
+            n = 3*n + 1;
+        }
+    }
+    return count;
```

Comment:
마지막 문장에는 `return`을 생략할 수 있습니다.
```suggestion
    count
```